deployment/ansible/services.yml

```diff
@@ -6,6 +6,9 @@
     - name: Update APT cache
       apt: update_cache=yes cache_valid_time=3600
 
+    - name: Install ACL utilities
```

Comment:
This package includes the `setfacl` tool, which Ansible 2.1+ uses to set POSIX ACLs to share modules with unprivileged users (`postgres` in this case).

Because setting `pipelining = True` didn't work during the first pass, I suspect that we can omit that and just stick with installing the `acl` package.

See: http://docs.ansible.com/ansible/become.html